Aptoide/aptoide-client-v8
# Feature/asv 262 app view card

**What does this PR do?**

   All preview app cards should be same. 
Unfortunately, it was not possible to use the same layout everywhere.
There are two layouts: with ads and witout ads.
Where are the cards?
- home (ads and other bundles)
- home + more of trending bundle
- home + more of ads bundle
- home + more of if snipe it bundle
- app view, press install => Similar app bundles has the two types of cards:

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppViewSuggestedAppDisplayable.java
- [ ] AppViewSuggestedAdDisplayable.java
- [ ] GridAppDisplayable.java
- [ ] GridAdDisplayable.java.java
- [ ] AppViewSuggestedAppWidget.java
- [ ] AppViewSuggestedAdWidget.java
- [ ] GridAppWidget.java
- [ ] GridAdWidget.java


**How should this be manually tested?**

Where are the cards?
- home (ads and other bundles)
- home + more of trending bundle
- home + more of ads bundle
- home + more of if snipe it bundle
- app view, press install => Similar app bundles has the two types of cards: with and without adds

Known problem:
- [ ] AppViewSuggestedAppDisplayable.java
- [ ] GridAppDisplayable.java
and
- [ ] AppViewSuggestedAdDisplayable.java
- [ ] GridAdDisplayable.java.java
we should be able to re-use the layout.xml, but we can't

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-296
https://aptoide.atlassian.net/browse/ASV-262


**Questions:**

   Does this add new dependencies which need to be added? No


**Code Review Checklist**
